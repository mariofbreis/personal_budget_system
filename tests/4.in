i 100 0 0 r Receita
i 200 1 0 r Receita
i 300 1 0 r Receita
i 400 0 0 r Receita
i 50 999 0 d Despesa
i 51 999 0 d Despesa
i 52 999 0 d Despesa
i 53 999 0 d Despesa
i 50 500 0 r Receita
i 50 500 0 d Despesa
b 0
b 1
b 999
b 500
b 100
