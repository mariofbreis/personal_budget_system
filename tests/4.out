500
1000
794
1000
1000
